ci-operator/config/openshift/openstack-test/openshift-openstack-test-main.yaml

```diff
@@ -88,6 +88,22 @@ tests:
     - ref: openstack-test-openstack
     workflow: openshift-e2e-openstack-ipi
   timeout: 6h0m0s
+- always_run: false
+  as: e2e-openstack-dualstack
+  optional: true
+  steps:
+    cluster_profile: openstack-hwoffload
+    workflow: openshift-e2e-openstack-dualstack
+- always_run: false
+  as: e2e-openstack-dualstack-v6primary
+  optional: true
+  steps:
+    cluster_profile: openstack-hwoffload
+    env:
+      CONFIG_TYPE: dualstack-v6primary
+      TEST_SKIPS: when FIPS is disabled the HAProxy router should serve routes when
```

Comment:
why do we skip this?

Reply:
those tests are failing on ocp e2e tests and were not debugged yet.